torch/cuda/amp/grad_scaler.py

```diff
@@ -422,12 +424,14 @@ def update(self, new_scale=None):
                 for i in range(1, len(found_infs)):
                     found_inf_combined += found_infs[i]
 
-            torch._amp_update_scale_(_scale,
```

Comment:
Could the stylistic changes go in a separate PR? This would make PRs clearer to review and make `git blame`'ing in the future more accurate.